Bookish/Controllers/MemberController.cs

```diff
@@ -0,0 +1,70 @@
+using System.Diagnostics;
+using Microsoft.AspNetCore.Mvc;
+using Bookish.Models;
+using Microsoft.AspNetCore.Authorization.Infrastructure;
+
+namespace Bookish.Controllers;
+
+public class MemberController : Controller
+{
+    public IActionResult MemberList()

```

Comment:
Similar to my comment in `BooksController`, this endpoint should have an annotation e.g. `[HttpGet]` for consistency